docs/config.rst

```diff
@@ -684,6 +863,69 @@ Full Default Configuration
             </doctrine:config>
         </container>
 
+    .. code-block:: php
+
+        use Symfony\Config\DoctrineConfig;
+
+        return static function (DoctrineConfig $config): void {
+            $config->autoGenerateHydratorClasses(0);
+            $config->autoGenerateProxyClasses(0);
+            $config->defaultConnection('');
+            $config->defaultDatabase('default');
+            $config->defaultDocumentManager('');
+            $config->hydratorDir('%kernel.cache_dir%/doctrine/odm/mongodb/Hydrators');
+            $config->hydratorNamespace('Hydrators');
+            $config->proxyDir('%kernel.cache_dir%/doctrine/odm/mongodb/Proxies');
+            $config->proxyNamespace('Proxies');
+            $config->fixtureLoader('Symfony\Bridge\Doctrine\DataFixtures\ContainerAwareLoader');
+
+            $config->documentManager('id')
+                ->connection('')
+                ->database('')
+                ->defaultDocumentRepositoryClass('')
+                ->defaultGridfsRepositoryClass('')
+                ->repositoryFactory('')
+                ->logging(true)
+                ->autoMapping(false)
+                ->metadataCacheDriver()
+                    ->type('')
+                    ->class('')
+                    ->host('')
+                    ->port('')
```

Comment:
This list of "defaults" is incorrect. There is no default values as this are not set by default.